webpack.config.js

```diff
@@ -0,0 +1,78 @@
+var webpack = require('webpack');
+var path = require('path');
+var NODE_ENV = process.env.NODE_ENV === 'Development';
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const extractSass = new ExtractTextPlugin({
+    filename: '[name].[contenthash].css',
+    disable: process.env.NODE_ENV === 'Development'
+});
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
+
+module.exports = {
+    context: __dirname,
+    entry: {
+        app: 'react-hot-loader/patch',
+        bundle: './src/index.js',
+        styles: './src/index.scss'
```

Comment:
It is better to treat index.scss as dependency of index.js, I believe. Try to remove this from webpack config and add `import styles from index.scss` in index.js